misc/package/Data Files/MWSE/core/mcm/components/settings/DecimalSlider.lua

```diff
@@ -36,29 +36,24 @@ function DecimalSlider:new(data)
 	return Parent.new(self, data) -- the `__index` metamethod will make the `min`, `max`, etc fields default to the values specified above.
 end
 
-function DecimalSlider:scaleToSliderRange(value)
-	return value * 10 ^ self.decimalPlaces
+
+function DecimalSlider:convertToSliderValue(variableValue)
+	return 10 ^ self.decimalPlaces * (variableValue - self.min)
 end
 
-function DecimalSlider:scaleToVariableRange(value)
-	return value / 10 ^ self.decimalPlaces
+function DecimalSlider:convertToVariableValue(sliderValue)
+	return self.min + sliderValue / 10 ^ self.decimalPlaces
 end
 
 function DecimalSlider:updateValueLabel()
```

Comment:
this is now called after `updateVariableValue`, so computation can now be done with `self.variable.value` instead of `self.elements.slider.widget.current`